README.md

```diff
@@ -1,262 +1,304 @@
+Using Pip is the recommended way to install the Learnosity SDK for Python in production. The easiest way is to run this from your parent project folder:
 
     pip install learnosity_sdk
 
-Or, from a checkout of this repo:
+### **Alternative method 1: download the zip file**
+Download the latest version of the SDK as a self-contained ZIP file from the [GitHub Releases](https://github.com/Learnosity/learnosity-sdk-python/releases) page. The distribution ZIP file contains all the necessary dependencies. 
 
-    pip install .
+Note: after installation, run this command in the SDK root folder:
 
-# Upgrading
+    pip install .
 
-If installed from pypi:
+### **Alternative 2: development install from a git clone**
+To install from the terminal, run this command:
 
-    pip install --upgrade learnosity_sdk
+    git clone git@github.com:Learnosity/learnosity-sdk-python.git
 
-Or just do a pull from GitHub and run again in the repo:
+Note: after installation, run this command in the SDK root folder:
 
     pip install .
 
-# Usage
+Note that these manual installation methods are for development and testing only.
+For production use, you should install the SDK using the Pip package manager for Python, as described above.
 
-## Questions API
+[(Back to top)](#table-of-contents)
 
-```python
-import learnosity_sdk.request
+## Quick start guide
+Let's take a look at a simple example of the SDK in action. In this example, we'll load an assessment into the browser.
 
-# Security packet including consumer key
-security = {
-  'consumer_key': 'MY_API_KEY',
-  'domain': 'localhost',
-  'user_id': '$ANONYMIZED_USER_ID'
-}
-# consumer secret for API access
-# WARNING: The consumer secret should not be committed to source control.
-secret = 'MY_API_SECRET'
-
-# request data for Questions API
-questions_request = {
-    "type": "local_practice", "state": "initial",
-    "questions": [
-        {
-            "response_id": "60005",
-            "type": "association",
-            "stimulus": "Match the cities to the parent nation",
-            "stimulus_list": [
-                "London", "Dublin", "Paris", "Sydney"
-            ],
-            "possible_responses": [
-                "Australia", "France",
-                "Ireland", "England"
-            ],
-            "validation": {
-                "valid_responses": [
-                    ["England"], ["Ireland"], ["France"], ["Australia"]
-                ]
-            }
-        }
-    ]
-}
+### **Start up your web server and view the standalone assessment example**
+To start up your Python web server, first find the following folder location under the SDK. Change directory ('cd') to this location on the command line.
 
-questions_init = learnosity_sdk.request.Init(
-    'questions', security, secret,
-    request=questions_request
-)
+If installed under Pypi, navigate to this location:
+
+    .../usr/local/lib/python3.9/site-packages/learnosity_sdk/docs/quickstart/assessment/
```

Comment:
Path here is python version specific... perhaps needs to be expressed in a more generic way

Reply:
How about this? Does that do the trick? How could it be better? 
/usr/local/lib/python3.x/site-packages/learnosity_sdk/docs/quickstart/assessment/

Reply:
Updated with a note explaining that the path is version dependent: https://github.com/Learnosity/learnosity-sdk-python/commit/0658d15f2afa5b8ebe032c0cdf9db228b0991578